### PR TITLE
Zsh

### PR DESCRIPTION
**Pull request by satyaog**

### CODE REVIEW

File: docs/Userguide_login.rst [hunk 1]
```diff
@@ -189,3 +189,11 @@ require a lot of resources: editing via VSCode, building conda environments, tes
...
 While it is possible to tweak it to allocate a GPU, doing so will prevent simultaneous
 connections to it (until Slurm is upgraded to version 22.05 or later).
+
+Using a non-Bash Unix shell
+---------------------------
+
+While IT and Infra will not provide personalized support for non-Bash Unix
+shell on the cluster, the cluster is designed to support both Bash and Zsh
+shells. If you think things should work with Zsh and they don't, please contact
+`Mila's IT support <https://it-support.mila.quebec>`_.
```
Comment (satyaog) [inline docs/Userguide_login.rst:199]: Not sure if I should include details on what is implied by "the cluster is designed to support both Bash and Zsh". I'm thinking of how the modules are loaded but I'm not sure if this should be mentioned and if there is more.

Comment (satyaog) [inline docs/Userguide_login.rst:199]: Not sure if this should be rephrased, I would like it and infra to give me some feedback as they would be directly involved in answering user

File: docs/Userguide_login.rst [hunk 2]
```diff
@@ -189,3 +189,11 @@ require a lot of resources: editing via VSCode, building conda environments, tes
 The ``mila-cpu`` entry should be in your ``.ssh/config``. Changes are at your own risk.
 While it is possible to tweak it to allocate a GPU, doing so will prevent simultaneous
 connections to it (until Slurm is upgraded to version 22.05 or later).
+
+Using a non-Bash Unix shell
+---------------------------
+
+While IT and Infra will not provide personalized support for non-Bash Unix
```
Comment (satyaog) [inline docs/Userguide_login.rst:196]: Would `1-on-1 support` be better? The idea is that users would not request help to setup their stuff with Zsh, but still know that it should work and for the team to know if things break

File: docs/Userguide_login.rst (hunk 1 above)
Comment (btravouillon) [reply docs/Userguide_login.rst:199]: I would got with Mila.
```suggestion
While Mila will not provide personalized support for non-Bash Unix
shell on the cluster, the cluster is designed to support both Bash and Zsh
shells. If you think things should work with Zsh and they don't, please contact
`Mila's IT support <https://it-support.mila.quebec>`_.
```

File: docs/Userguide_login.rst (hunk 2 above)
Comment (satyaog) [reply docs/Userguide_login.rst]: Message updated

File: docs/Userguide_login.rst [hunk 3]
```diff
@@ -189,3 +189,11 @@ require a lot of resources: editing via VSCode, building conda environments, tes
...
 While it is possible to tweak it to allocate a GPU, doing so will prevent simultaneous
 connections to it (until Slurm is upgraded to version 22.05 or later).
+
+Using a non-Bash Unix shell
+---------------------------
+
+While Mila does not provide support in debugging your shell setup, Bash is the
+standard shell to be used on the cluster and the cluster is designed to support
+both Bash and Zsh shells. If you think things should work with Zsh and they
+don't, please contact `Mila's IT support <https://it-support.mila.quebec>`_.
```
Comment (satyaog) [inline docs/Userguide_login.rst:199]: Not sure if I should include details on what is implied by "the cluster is designed to support both Bash and Zsh". I'm thinking of how the modules are loaded but I'm not sure if this should be mentioned and if there is more.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Review by priscillaben — APPROVED** [review]

**Pull request merged**